src/method-dispatch.c

```diff
@@ -4,6 +4,38 @@
 
 extern SEXP parent_sym;
 extern SEXP sym_ANY;
+extern SEXP ns_S7;
+extern SEXP sym_obj_dispatch;
+extern SEXP sym_dispatch_args;
+extern SEXP sym_methods;
+extern SEXP fn_base_quote;
+extern SEXP fn_base_missing;
+
+extern SEXP R_TRUE;
+
+
+static inline
+void APPEND_NODE(SEXP node, SEXP val, SEXP tag) {
```

Comment:
I would've put `tag` before `val`? Not sure why, but that ordering feels more logical to me.